helm/cert-exporter/templates/daemonset.yaml

```diff
@@ -30,7 +30,7 @@ spec:
         - --monitor-secrets={{ .Values.config.daemonset.monitorSecrets }}
         - --monitor-certificates={{ .Values.config.daemonset.monitorCertificates }}
         - --monitor-files={{ .Values.config.daemonset.monitorFiles }}
-        - --cert-paths={{ default "/etc/kubernetes/ssl" .Values.exporter.certPath }}
+        - --cert-paths={{ default "/etc/kubernetes/ssl,/etc/kubernetes/pki" .Values.exporter.certPath }}
```

Comment:
This changes the default but we are passing a value in the `config` repo https://github.com/giantswarm/config/blob/main/default/apps/cert-exporter/configmap-values.yaml.template#L2

Maybe we can delete it from the `config` repo and rely on the default?

Reply:
That'd work too. I just thought it would be less code. But both should work.

Reply:
I just meant maybe keeping it in the config repo makes it more discoverable, but yeah, deleting it from there also works.